From 87f70c6a1d3504e34bbd9c53476f41adcebc1b8f Mon Sep 17 00:00:00 2001
From: Sudarshan Bisht <sudarshan.bisht@nokia.com>
Date: Fri, 30 Sep 2011 12:49:54 +0300
Subject: [PATCH 111/125] null-sink: Set latency range at the time of
 initialization of module.

At the time of module initialization latency range is being set so that the null-sink
would be aware of its limitations with latencies.
---
 src/modules/module-null-sink.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/modules/module-null-sink.c b/src/modules/module-null-sink.c
index 10868f5..454bd3e 100644
--- a/src/modules/module-null-sink.c
+++ b/src/modules/module-null-sink.c
@@ -315,6 +315,8 @@ int pa__init(pa_module*m) {
         goto fail;
     }
 
+    pa_sink_set_latency_range(u->sink, 0, BLOCK_USEC);
+
     pa_sink_put(u->sink);
 
     pa_modargs_free(ma);
-- 
1.7.5.4

